pyproject.toml

```diff
@@ -26,7 +26,8 @@ dynamic = ["version"]
 dependencies = [
     "gitpython>3",
     "dulwich>=0.24.0",
-    "pygit2>=1.14.0",
+    "pygit2>=1.14.0,<1.19.0; python_version < '3.11'",
+    "pygit2>=1.19.0; python_version >= '3.11'",
```

Comment:
Do we need this at all? `pip` would likely not install the new versions for old python versions because `pygit2` has properly setup `requires-python` (I hope?).

Reply:
ok, let's drop this as the above code will gracefully fallback in case old version is installed.

Reply:
done